index.js

```diff
@@ -0,0 +1,43 @@
+const fs = require('fs');
+const { spawnSync } = require('child_process');
+const os = require('os');
+const readline = require('readline');
+const { getCurrentUnixTime, clearLine, getComand,logFileName, windowsCommand, unixLikeOSCommand, timeToRefresh } = require('./helpers');
+
+if (!fs.existsSync(logFileName)) {
+  fs.writeFileSync(logFileName, '');
+}
```

Comment:
You are using appendFile API to add data to the file, and it creates the file automatically if it doesn't exist. So this code is redundant.

Reply:
Femoved, thanks.